tests/fedora-str/sysinstall-tests.yml

```diff
@@ -0,0 +1,23 @@
+---
+- hosts: localhost
+  tags:
+  - atomic
+  remote_user: root
+  tasks:
+  - find:
+      paths: /root/tests/installed/
+      patterns: "itest-*.sh"
+    register: installed_tests
+  - name: Run sysinstalled tests
+    shell: /root/tests/installed/run.sh &> /root/installed-tests.log
```

Comment:
This is OK, though things will get tricky when we want to e.g. have tests that reboot/modify the VM. It could be interesting to have a hybrid approach of this and rpm-ostree's `libvm` where tests can seamlessly use the ansible infra for running code remotely, rebooting, pushing/fetching files, etc... Or do you already have an idea in mind on how to approach this?

Reply:
Right, I think the next step is to change some of the installed tests to be playbooks.